### ISSUE TITLE
No active editor on Ubuntu 64 bit

### ISSUE THREAD

**Issue by superfine** [emmetio/livestyle-sublime-old#60]
Hey, LiveStyle was working fine last week, no problems or errors with Sublime Text 2.

Now I am getting the "No active editor" Error like in Issues #56 and #40.

**Sublime Version 2.0.2, Build 2221
Emmet LiveStyle 0.8.1.819
Chrome Version 28.0.1500.71 Ubuntu 12.04 (28.0.1500.71-0ubuntu1.12.04.1)
Ubuntu 12.04 64bit**

Console Log

```
startup, version: 2221 linux x64 channel: stable
executable: /opt/bin/Sublime Text 2/sublime_text
working dir: /home/plaubner
packages path: /home/plaubner/.config/sublime-text-2/Packages
settings path: /home/plaubner/.config/sublime-text-2/Settings
PackageSetup not required
Py_GetProgramName(): sublime_text
Py_GetExecPrefix(): .
Py_GetProgramFullPath(): 
Py_GetPath(): ./lib/python26.zip:./lib/python2.6/:./lib/python2.6/plat-linux2:./lib/python2.6/lib-tk:./lib/python2.6/lib-old:./lib/python2.6/lib-dynload
Py_GetPythonHome(): .
catalogue loaded
found 7 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 4 files for base name Main.sublime-menu
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/CSS/css_completions.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/ColorPicker/sublimecp.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/comment.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/copy_path.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/delete_word.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/detect_indentation.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/duplicate_line.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/echo.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/exec.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/fold.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/font.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/goto_line.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/indentation.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/kill_ring.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/mark.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/new_templates.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/open_file_settings.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/open_in_browser.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/paragraph.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/save_on_focus_lost.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/scroll.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/set_unsaved_view_name.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/side_bar.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/sort.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/swap_line.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/switch_file.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/transform.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/transpose.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Default/trim_trailing_white_space.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Diff/diff.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Emmet/emmet-plugin.py
Emmet: No need to update PyV8
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/add.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/annotate.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/commit.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/diff.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/flow.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/git.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/history.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/repo.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/stash.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/status.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Git/statusbar.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/HTML/encode_html_entities.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/HTML/html_completions.py
Reloading plugin /home/plaubner/.config/sublime-text-2/Packages/Package Control/Package Control.py
plugin init time: 0.0998
loading bindings
loading pointer bindings
found 1 files for base name Default.sublime-theme
theme loaded
app ready
pre session restore time: 0.146404
using gamma: 1 (err: 0)
startup cache, total files: 105 cache hits: 105
startup time: 0.222225 (package setup was not run)
loaded 887 snippets
Package Control: Skipping automatic upgrade, last run at 2013-08-19 14:34:22, next run at 2013-08-19 15:34:22 or after

```

I also tested Sublime Text 3. In Chrome the Sublime Text 3 Editor is shown but no changes are pushed.

```
DPI scale: 1
startup, version: 3047 linux x64 channel: stable
executable: /opt/sublime_text/sublime_text
working dir: /
packages path: /home/plaubner/.config/sublime-text-3/Packages
state path: /home/plaubner/.config/sublime-text-3/Local
hardware concurrency: 4
zip path: /opt/sublime_text/Packages
zip path: /home/plaubner/.config/sublime-text-3/Installed Packages
found 3 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 3 files for base name Main.sublime-menu
loading bindings
loading pointer bindings
found 1 files for base name Default.sublime-theme
theme loaded
app ready
wrote startup cache, added files: 1 orphaned files: 0 total files: 152 cache hits: 151
pre session restore time: 0.076302
using gamma: 1 (err: 0)
startup time: 0.13156
first paint time: 0.148697
launching: /opt/sublime_text/plugin_host
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin LiveStyle.livestyle
loaded 837 snippets
reloading plugin Package Control.Package Control
plugins loaded
Package Control: Skipping automatic upgrade, last run at 2013-08-19 15:02:58, next run at 2013-08-19 16:02:58 or after
Emmet LiveStyle: No suitable client for diff
Emmet LiveStyle: No suitable client for diff
Emmet LiveStyle: No suitable client for diff

```

Help?

**Comment by sergeche** [comment]
You should update Chrome extension

**Comment by sergeche** [comment]
And I don’t see installed LiveStyle in your ST2 (according to log)

**Comment by superfine** [comment]
My Chrome Extension is **0.8.1.819**, which is the lastest or is there a newer Version?

**Comment by superfine** [comment]
ST2: Package Control.sublime-settings

```
{
    "installed_packages":
    [
        "ColorPicker",
        "CSS Snippets",
        "Emmet",
        "Future Funk - Color Scheme",
        "Git",
        "LiveStyle",
        "Package Control",
        "Zissou Color Schemes"
    ]
}
```

**Comment by sergeche** [comment]
Try to restart Chrome.

In your ST2 log, I don’t see that LiveStyle is actually loaded, it’s not in plugin loaded list

**Comment by superfine** [comment]
I'm gonna do a clean Install of Sublime Text 2 and LiveStyle, without any other Plugins, be right back

**Comment by superfine** [comment]
Now Sublime and LiveStyle are working but sometimes I still get an Error: **Emmet LiveStyle: [client] No associated file found**.

I have no other Plugins installed at the moment though.

**Comment by sergeche** [comment]
This message means you didn’t associated files in LiveStyle panel

**Comment by superfine** [comment]
I get the Error even if the File mapping is set to **main.css = main.css**. I think there is a conflict between Plugins. Test Sublime Text 3 later. Thanks for your help, need to get back to my code ;-)

**Comment by sergeche** [comment]
No, it can’t be a plugin conflict. The `No associated file found` means you’re make updates in file that have no associations. Can you describe how you make changes?

**Comment by superfine** [comment]
I have Sublime open on one monitor with the CSS I want to edit and Chrome in another Monitor with the enable LiveStyle screen open in the Inspector. I start Sublime always first then Chrome.

Here is another Error while working just now:

```
A plugin (livestyle) may be making Sublime Text unresponsive by taking too long (0.046846s) in its on_modified callback.

This message can be disabled via the detect_slow_plugins setting
```

**Comment by sergeche** [comment]
Can you describe how _exactly_ you modify CSS? Update value in Inspector tab, in editor, created new selector in inspector etc.

The _slow plugin_ message is not an error, it’s just a notification of ST2 that some operations takes too much time (in this case, 46ms) to execute. You can simply disable this notification as described.

**Comment by superfine** [comment]
I work, thanks to LiveStyle now, fully bi-directional.

In Sublime I add classes and edit selectors and values, in the Inspector I do the fine tuning on values but add new selectors sometimes. I hardly had problems with the actual use of the plugin and extension so far.

Sometimes I have to reload the web page (because of editing the PHP/YML files) but I have no problems usually and if had, I just relinked the correct CSS file.

I have very many tabs (30+) open though, I have closed a lot of them in the mean time and LiveStyle seems to running smoothly.

Currently the only plugins installed in Sublime 2 are:

```
{
    "installed_packages":
    [
        "ColorPicker",
        "CSS Snippets",
        "Future Funk - Color Scheme",
        "LiveStyle",
        "Package Control",
        "Zissou Color Schemes"
    ]
}
```

Tomorrow I'll re-install the rest and see If I have issues again.
